Documentation/General/finalReport/overview.tex

```diff
@@ -1 +1,40 @@
-\chapter{Overview of the developed and implemented software product}
\ No newline at end of file
+\chapter{Overview of the developed and implemented software product}
+
+\section{Introduction}
+When entering MoodCat, a user can choose multiple of a list of moods to listen music to. Based on these moods a list of rooms
+sorted on relevance is shown, along with its title and which song is currently playing. When entering a room
+the user can chat with other users that have joined the room.
```

Comment:
In my part I made a small overview with a more broath view of the system.
Maybe that should be in this part.

I think it should be excluded from my chapther, so there is no duplication.

We should discuss that tommorow, I think.